dll/win32/shell32/shlexec.cpp

```diff
@@ -49,6 +49,19 @@ static BOOL SHELL_InRunDllProcess(VOID)
     return s_bInDllProcess;
 }
 
+static UINT_PTR InvokeOpenWith(HWND hWndOwner, SHELLEXECUTEINFOW &sei)
+{
+    extern HRESULT SH32_InvokeOpenWith(PCWSTR, LPCMINVOKECOMMANDINFO, HANDLE *);
+
+    HANDLE *phProc = (sei.fMask & SEE_MASK_NOCLOSEPROCESS) ? &sei.hProcess : NULL;
+    UINT fCmic = (sei.fMask & SEE_CMIC_COMMON_BASICFLAGS) | CMIC_MASK_FLAG_NO_UI;
+    CMINVOKECOMMANDINFO ici = { sizeof ici, fCmic, hWndOwner };
+    ici.nShow = SW_SHOW;
+    HRESULT hr = SH32_InvokeOpenWith(sei.lpFile, &ici, phProc);
+    SetLastError(ERROR_NO_ASSOCIATION);
+    return SUCCEEDED(hr) ? 42 : SE_ERR_NOASSOC;
```

Comment:
Here, the "42" is documented on MSDN, or consistent (i.e. the same) as in other similar success codes returned in other shell-execute functions ?